pkg/app/app.go

```diff
@@ -96,6 +96,16 @@ func info() *Data {
 		environment = env
 	}
 
+	var domain string
+	switch environment {
+	case "development":
```

Comment:
This is open source, so storing these here doesn't really work. I'd suggest a different package that consumes `app.Info()` and returns these values as required.

Reply:
`ServiceID`, realistically, shouldn't have been put here either.

My other concern is also with mixing authn and application information, which I think should be owned entirely by an authn specific package only (as I somewhat implied above)

Reply:
ok, in that case, i will move this into mint, and will close this pr. thx.